src/simplewallet/simplewallet.cpp

```diff
@@ -3691,6 +3805,17 @@ bool simple_wallet::show_transfers(const std::vector<std::string> &args_)
     }
   }
 
+  // subaddr_index
+  std::set<uint32_t> subaddr_indices;
+  if (local_args.size() > 0 && local_args[0].substr(0, 6) == "index=")
+  {
+    if (!parse_subaddress_indices(local_args[0], subaddr_indices))
+      return true;
+    in = pool = true;
+    out = pending = failed = false;
+    local_args.erase(local_args.begin());
+  }
+
```

Comment:
I think the logic is exactly the same as with the first optional parameter `[in|out|all|pending|failed]`; if the argument is nonsensical, it's skipped and passed to the subsequent check for the `min_height` where it errors out with `"bad min_height parameter"`.